packages/cubejs-schema-compiler/src/compiler/ErrorReporter.ts

```diff
@@ -141,16 +141,16 @@ export class ErrorReporter {
     return this.rootReporter().errors;
   }
 
-  public addErrors(errors: CompilerErrorInterface[]) {
-    this.rootReporter().errors.push(...errors);
+  public addErrors(errors: any[]) {
```

Comment:
`any` is not an option for a new code. Next engineer will reverse engineer code just to know the type. No validation.

Reply:
Resolved in https://github.com/cube-js/cube/pull/9234